nexmark-flink/src/main/resources/queries-cep/q0.sql

```diff
@@ -0,0 +1,37 @@
+-- -------------------------------------------------------------------------------------------------
+-- Query 0: Periods of a constantly decreasing price (Not in original suite)
+-- -------------------------------------------------------------------------------------------------
+-- Finds periods of a constantly decreasing price of a single bidder.
+-- Illustrates a typical Pattern in MATCH_RECOGNIZE.
+-- -------------------------------------------------------------------------------------------------
+
+CREATE TABLE discard_sink (
+      bidder  BIGINT,
+      start_tstamp  TIMESTAMP(3),
+      bottom_tstamp  TIMESTAMP(3),
+      end_tstamp  TIMESTAMP(3)
+) WITH (
+    'connector' = 'blackhole'
+);
+
+INSERT INTO discard_sink
+SELECT
+    bidder, start_tstamp, bottom_tstamp, end_tstamp
+FROM bid
+MATCH_RECOGNIZE (
+    PARTITION BY bidder
```

Comment:
Makes more sense to `PARTITION BY bidder, auction`?

Reply:
@dianfu, why does `PARTITION BY bidder, auction` make more sense? The queries are only oriented to bidder and don't use the `auction`.